gamepanel/game.php

```diff
@@ -194,6 +194,72 @@ function gameIcons()
 		return $buf;
 	}
 
+	/*
+	function phaseSwitchInfo()
+	{
+		$buf = '';
+		
+		if ($this->phase == 'Finished' or $this->phaseSwitchPeriod <= 0){
+			return $buf;
+		}
+		
+		$buf .= '<div class="titleBarRightSide">'.
+					l_t('Next phase length: <span class="nextPhaseMinutes"><strong>%s</strong> /phase</span>', libTime::timeLengthText($this->nextPhaseMinutes * 60)).'</div>';
+			
+		if ($this->startTime > 0){
+			
+			$timeWhenSwitch = (($this->phaseSwitchPeriod * 60) + $this->startTime);
+			
+			
+			$timeWhenSwitch = libTime::remainingText($timeWhenSwitch);
+			
+			$buf .= '<div style="clear:both"></div>';
+			$buf .= '<div class="titleBarRightSide">'.
+						l_t('Phase switch: <strong>%s</strong>', $timeWhenSwitch).'</div>';	
+		}
+		
+		else {
+			$timeTillNextPhase = libTime::timeLengthText($this->phaseSwitchPeriod * 60);
+			
+			$buf .= '<div style="clear:both"></div>';
+			$buf .= '<div class="titleBarRightSide">'.
+						l_t('Phase switch: <strong>%s</strong> after game start', $timeTillNextPhase).'</div>';	
+		}
+								
+
+		return $buf;
+	}
+	*/
+	
+	function phaseSwitchInfo()
+		{

```

Comment:
This isn't a huge deal but just from a readability standpoint, we try and keep the indentation of { at the same as the function definition line. 
so

function phaseSwitchInfo()
{

Reply:
Donezo!

Reply:
Thanks!